devicetypes/sensative/sensative-strips-guard-700.src/sensative-strips-guard-700.groovy

```diff
@@ -0,0 +1,482 @@
+	try {
+		if (zwaveInfo?.zw?.contains("s") || ("0x98" in device?.rawDescription?.split(" "))) {
+			return zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+		}
+		else {
+			return cmd.format()
+		}
+	}
+	catch (ex) {
+		return cmd.format()
+	}
+}
+
+void sendCommands(List<String> cmds, Integer delay=100) {
+	if (cmds) {
+		def actions = []
+		cmds.each {
+			actions << new physicalgraph.device.HubAction(it)
+		}
+		sendHubCommand(actions, delay)
+	}
+}
+
+
+def parse(String description) {
+	def cmd = zwave.parse(description, commandClassVersions)
+	if (cmd) {
+		zwaveEvent(cmd)
+	}
+	else {
+		log.warn "Unable to parse: $description"
+	}
+
+	updateLastCheckIn()
+	return []
+}
+
+void updateLastCheckIn() {
+	if (!isDuplicateCommand(state.lastCheckInTime, 60000)) {
+		state.lastCheckInTime = new Date().time
+
+		sendEvent(name: "lastCheckIn", value: convertToLocalTimeString(new Date()), displayed: false)
+	}
+}
+
+String convertToLocalTimeString(dt) {
+	try {
+		def timeZoneId = location?.timeZone?.ID
+		if (timeZoneId) {
+			return dt.format("MM/dd/yyyy hh:mm:ss a", TimeZone.getTimeZone(timeZoneId))
+		}
+		else {
+			return "$dt"
+		}
+	}
+	catch (ex) {
+		return "$dt"
+	}
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCmd = cmd.encapsulatedCommand(commandClassVersions)
+	if (encapsulatedCmd) {
+		zwaveEvent(encapsulatedCmd)
+	}
+	else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+	}
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.wakeupv2.WakeUpNotification cmd) {
+	logDebug "Device Woke Up..."
+
+	List<String> cmds = []
+
+	cmds += getConfigureCmds()
+
+	if (cmds) {
+		cmds << "delay 500"
+	}
+	else {
+		cmds << batteryGetCmd()
+	}
+
+	cmds << wakeUpNoMoreInfoCmd()
+	sendCommands(cmds)
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.wakeupv2.WakeUpIntervalReport cmd) {
+	logDebug "Wake Up Interval = ${cmd.seconds} seconds"
+	state.wakeUpInterval = cmd.seconds
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.versionv1.VersionReport cmd) {
+	logTrace "${cmd}"
+	
+	sendEventIfNew("firmwareVersion", (cmd.applicationVersion + (cmd.applicationSubVersion / 100)))
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
+	int val = (cmd.batteryLevel == 0xFF ? 1 : safeToInt(cmd.batteryLevel))
+	if (val > 100) val = 100
+	if (val < 1) val = 1
+
+	String desc = "${device.displayName}: battery is ${val}%"
+	logDebug(desc)
+
+	sendEvent(name: "battery", value: val, unit: "%", isStateChange: true, descriptionText: desc)
+}
+
+
+void zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	logTrace "${cmd}"
+
+	runIn(4, refreshSyncStatus)
+
+	Map param = configParams.find { it.num == cmd.parameterNumber }
+	if (param) {
+		logDebug "${param.name}(#${param.num}) = ${cmd.scaledConfigurationValue}"
+		setParamStoredValue(param.num, cmd.scaledConfigurationValue)
+	}
+	else {
```

Comment:
Please remove this line break before the "else" statement.

Reply:
Removed